homeassistant/components/uptimerobot/strings.json

```diff
@@ -5,13 +5,19 @@
         "description": "You need to supply the 'main' API key from UptimeRobot",
         "data": {
           "api_key": "[%key:common::config_flow::data::api_key%]"
+        },
+        "data_description": {
+          "api_key": "The API key for the service"
```

Comment:
I think using data_description can be used to remove the main description (and also the `'main' API key` mention)

Reply:
There are at least 2 different key types and we need the "main" one for the integration to work

Reply:
Yes, but I mean, we can explain that in the data_description instead of the main description